agent/tool-scripts/node-exporter

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/python3
+# -*- mode: python -*-
+
+import os
+import sys
+import logging
+
+PROG = os.path.basename(sys.argv[0])
+logger = logging.getLogger(PROG)
+logger.setLevel(logging.DEBUG)
+sh = logging.StreamHandler()
+sh.setLevel(logging.DEBUG)
+shf = logging.Formatter("%(message)s")
+sh.setFormatter(shf)
+logger.addHandler(sh)
+
+if len(sys.argv) != 2 or sys.argv[1] != "--help":
+    logger.info("This script is deprecated, please run it with --help for info on registering the tool.")
+    logger.info("Run /opt/pbench-agent/tool-scripts/node-exporter --help for more info.")
+    exit(0)
+
+if sys.argv[1] == "--help":
+    logger.info("Options:")
+    logger.info(
+        "--inst=</path/to/node_exporter/dir> (will append /node_exporter for exe, required)"
```

Comment:
What exactly is required here?

Reply:
One of the required options is the path to the node_exporter installation directory

Reply:
These tools, `dcgm` and `node-exporter`, are not actually invoked when run by the Tool Meister.

We really need to drop the tool-scripts "tool" symlinks to `base-tools` and replace with python modules as described in #1786.

Reply:
If they are going to be dropped, it does not matter. If there is a possibility that a human will see the message, I think it should be reworded:

```
--inst=</path/to/node_exporter/dir> (required; to create the path of the executable,  "/node-exporter" will be appended )
```
or something like that.